src/runner/browser-env/vite/utils.ts

```diff
@@ -23,6 +26,33 @@ export const getNodeModulePath = async ({
     return resolve(moduleName, path.join(rootFileUrl, parent));
 };
 
+// Just copy-pasted from modern-node-polyfills to avoid potential errors
+export const polyfillPath = async (moduleName: string): Promise<string> => {
```

Comment:
If this was taken from a MIT-licensed repo, you are required to include the corresponding license. For instance, you can create a separate file where in the beginning you copy-paste modern-node-polyfills' license to comment and then this code.

For our own ease of use, I'd also include a permalink to source file on github.